api/v1alpha1/clusterextension_types.go

```diff
@@ -107,14 +108,15 @@ type ServiceAccountReference struct {
 type PreflightConfig struct {
 	//+kubebuilder:Required
 	// CRDUpgradeSafety holds necessary configuration for the CRD Upgrade Safety preflight checks.
-	CRDUpgradeSafety *CRDUpgradeSafetyPreflightConfig `json:"crdUpgradeSafety,omitempty"`
+	CRDUpgradeSafety *CRDUpgradeSafetyPreflightConfig `json:"crdUpgradeSafety"`
 }
 
 // CRDUpgradeSafetyPreflightConfig is the configuration for CRD upgrade safety preflight check.
 type CRDUpgradeSafetyPreflightConfig struct {
 	//+kubebuilder:Required
-	// Disabled represents the state of the CRD upgrade safety preflight check being disabled/enabled.
-	Disabled bool `json:"disabled,omitempty"`
+	// +kubebuilder:validation:Enum:="Enabled","Disabled"
+	// policy represents the state of the CRD upgrade safety preflight check. Allowed values are "Enabled", and Disabled".
+	Policy CRDUpgradeSafetyPolicy `json:"policy"`
```

Comment:
Since we are using an Enum now, we should probably set the default value as `Enabled`.

We also need to update the code for the preflight checks here:
https://github.com/operator-framework/operator-controller/blob/1fb6bd28f917b881fcdd8fdc6ef512b157eebd94/internal/applier/helm.go#L72-L78

Reply:
I've already updated the code for the preflight check skipping logic in this commit. I'll set the default value as Enabled now.